.travis.yml

```diff
@@ -1,4 +1,7 @@
 sudo: false
+cache:
+  directories:
+    - $HOME/.cache/pip
```

Comment:
It's not if you override the default install section. See http://docs.travis-ci.com/user/caching/#pip-cache